packages/loader/driver-utils/src/networkUtils.ts

```diff
@@ -32,5 +39,5 @@ export function logNetworkFailure(
 
 	// non-retryable errors are fatal and should be logged as errors
 	newEvent.category = canRetryOnError(error) ? "generic" : "error";
-	logger.sendTelemetryEvent(newEvent, error);
+	logger.sendTelemetryEvent(newEvent, error as Error | undefined);
```

Comment:
I wonder if we ever call `logNetworkFailure` with something that is clearly not an `Error` or if we can tighten its signature. If we can't, let's at least leave a comment here that we added the cast to keep existing behavior when fixing linter rules but we might want to revisit.